### ISSUE TITLE
where is the loss of the sequence labeling?

### ISSUE THREAD

**Issue by jbjeong91** [WindChimeRan/CopyMTL#10]
Hello,
I can't find the loss of the sequence labeling in your code.
I can find only the loss for the decoder as follows:
-------------------------------------------------------------------
        loss = 0
        for t in range(self.seq2seq.decoder.decodelen):
            loss = loss + self.loss(pred_logits_list[t], triplets[:, t])
-------------------------------------------------------------------

**Comment by WindChimeRan** [comment]
Please see my new repo
https://github.com/WindChimeRan/OpenJERE

**Comment by jbjeong91** [comment]
Thanks for your reply. I found the loss for the 'BIO tag' in the Open JERE. Then, in this CopyMTL research "Extracting Relational Facts by an End-to-End Neural Model with Copy Mechanism", all of the scores be got without sequence labeling?

**Comment by WindChimeRan** [comment]
I got the score **with** sequence labeling. The original sequence labeling was very messy so I did not release it. The new version in OpenJERE uses a different implementation of CRF. The score will be similar.

**Comment by jbjeong91** [comment]
Then, is there a way to test the new CopyMTL model without UMTree?

**Comment by WindChimeRan** [comment]
Absolutely! You can run any models in OpenJERE independently!

**Comment by jbjeong91** [comment]
Thanks. and can I ask the last question?
In the paper of OpenJERE, the F1 score of the CopyMTL(NYT F1: 0.66) is lower than the score in CopyMTL's paper(NYT F1: 0.72).
What makes the difference between them?

**Comment by WindChimeRan** [comment]
Maybe it is the CRF, hyperparameters, or bugs in the original implementation. The newer version is clearer and less likely to have any bugs.

You can compare your model with the new score.

**Comment by jbjeong91** [comment]
Hello, I'm sorry to bother you.
Can I get information about the pre-processed NYT data you released?. I can see there are one 1-dimension list and two 2-dimension lists.  I just infer two 2-dimension lists are for sentence and triplets(head- tail-relation order). and I can't infer the 1-dimension list's role. Could you explain the role of each list?

**Comment by WindChimeRan** [comment]
I get the dataset from https://github.com/xiangrongzeng/copy_re, origin dataset. I don't get what you mean by 1-dim list.
